config/routes.rb

```diff
@@ -16,4 +16,11 @@
     resources :comments, only: [:create]
   end
 
+  resources :posts, only: [:index, :show] do
+    resources :votes, only: [:create]
+  end
+
+  resources :comments, only: [] do
+    resources :votes, only: [:create]
+  end
```

Comment:
You already have post resources and comment resources, so just nest votes in there.